### PR TITLE
Fix conda build file

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

**Review by wvreeven — APPROVED** [review]
LGTM

File: conda/meta.yaml [hunk 1]
```diff
@@ -14,10 +14,11 @@ build:
...
 test:
   requires:
-    - ts-conda-build
+    - ts-conda-build ==0.4
+    - ts-idl {{ idl_version }}
     - ts-xml
     - ts-salobj {{ salobj_version }}
-    - pymodbus >=3.6
+    - pymodbus ==3.7.2
     - pyserial-asyncio
```
Comment (couger01) [inline conda/meta.yaml]: This is part of pymodbus right so we don't need to specify it here.

File: conda/meta.yaml [hunk 2]
```diff
@@ -37,12 +38,12 @@ requirements:
...
+    - ts-conda-build ==0.4
   run:
     - python {{ python }}
     - setuptools
     - setuptools_scm
     - ts-xml
     - ts-salobj {{ salobj_version }}
-    - pymodbus >=3.6
+    - pymodbus ==3.7.2
     - pyserial-asyncio
```
Comment (couger01) [inline conda/meta.yaml]: Same as the comment above.

**Review by couger01 — APPROVED** [review]
Two recommendations otherwise LGTM.

**Pull request merged**